drizzlepac/haputils/generate_custom_svm_param_file.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
```

Comment:
A module level docstring explaining how to generate the custom parameter file, and what to expect for output, would be helpful.  It could show actual syntax for using the code, so an end-user could have a potentially working example to look at when trying to generate their own custom parameter file.  This docstring could also explain why it may be needed so that the user can see how it might apply to their reprocessing efforts.

Reply:
Issue resolved by commit [692003e](https://github.com/spacetelescope/drizzlepac/pull/896/commits/692003e8382804f997145e7093acda74e827fd7e)